src/lsst/cmservice/web_app/templates/pages/group_details.html

```diff
@@ -155,6 +165,11 @@
 <div class="w-full" id="group_content"
      hx-get="{{ url_for('get_group', campaign_id=group.campaign_id, step_id=group.step_id, group_id=group.id) }}"
      hx-trigger="every 20s" hx-select="#group_content" hx-swap="outerHTML">
+    <script>
+        groupScripts = {{scripts|sort(attribute="id")|tojson}};
+        groupJobs = {{jobs|sort(attribute="id")|tojson}};
+        wmsReport = {{group.wms_report|tojson}};
+    </script>
```

Comment:
Is this here to solve some kind of HTMX bug? Is there a reason this isn't at the top of the file? It would mean you don't need to set them to null first.

Reply:
Exactly! The grid's content would not refresh unless these data objects are within the section that's being auto-refreshed